parallel/task.go

```diff
@@ -277,11 +282,6 @@ func (p *ParallelTasks) Wait() ([]interface{}, error) {
 		return nil, nil
 	}
 
-	go func() {
-		p.wg.Wait()
-		p.done(nil)
-	}()
-
```

Comment:
This may cause channel leak
<img width="546" alt="image" src="https://github.com/user-attachments/assets/6dd40a6a-8eeb-4021-867c-90f3dd1c2bc6">